.circleci/config.yml

```diff
@@ -106,14 +106,7 @@ jobs:
           command: mkdir -p /tmp/artifacts
       - run:
           name: Build chain binary
-          command: ignite chain build -o /tmp/artifacts --clear-cache
-      - run:
-          name: Ensure ignite has not generated changes which have not been commited to VC
```

Comment:
I think we still want this step since it also catches if there are changes to the .proto files that haven't been built.

Reply:
maybe just do a proto-gen stage then